conf/medair.config

```diff
@@ -0,0 +1,46 @@
+//Profile config names for nf-core/configs
+params {
+  config_profile_description = 'Cluster profile for medair (local cluster of Clinical Genomics Gothenburg)'
+  config_profile_contact     = 'Clinical Genomics, Gothenburg'
```

Comment:
It would be good to have this a bit more specific. Either a specific person who runs nf-core pipelines regularly, or if you mean e.g. IT department (if they are maintaining the profile)

Reply:
Agree. [Clinical Genomics Stockholm](https://github.com/nf-core/configs/blob/master/conf/hasta.config) did the same... It is in the tutorial, but maybe we should also clarify it in the repo readme?

Reply:
Hm, I guess that should be updated...

https://github.com/nf-core/nf-co.re/pull/1225 for improving this in the tutorial at least.